samples/cpp/benchmark_app/main.cpp

```diff
@@ -320,12 +320,43 @@ int main(int argc, char* argv[]) {
         if (if_auto || if_multi) {
             devices.clear();
             std::string virtual_device = if_auto ? "AUTO" : "MULTI";
+            // check if device name specified by -d contains both AUTO and MULTI, like AUTO:MULTI,xxx, or like
+            // MULTI:AUTO,xxx.
+            if (if_auto && if_multi)
+                virtual_device = split(device_name, ':').at(0);
             auto iter_virtual = std::find(hardware_devices.begin(), hardware_devices.end(), virtual_device);
             hardware_devices.erase(iter_virtual);
             devices.push_back(virtual_device);
             parse_value_for_virtual_device(virtual_device, device_nstreams);
             parse_value_for_virtual_device(virtual_device, device_infer_precision);
         }
+
+        auto setDeviceProperty = [&core](std::string& device,
```

Comment:
I think this should be a regular function. There is no reason to define it as lambda